packages/astro/src/prerender/utils.ts

```diff
@@ -6,7 +6,7 @@ export function isServerLikeOutput(config: AstroConfig) {
 }
 
 export function getPrerenderDefault(config: AstroConfig) {
-	return config.output === 'hybrid';
+	return config.output !== 'server';
```

Comment:
What's the reason that this changed?

Reply:
The warnings were incorrect because static builds were being considered as prerendered by defauly